test/tbb/test_limiter_node.cpp

```diff
@@ -445,7 +445,14 @@ void test_try_put_without_successors() {
     int try_put_num{3};
     tbb::flow::buffer_node<int> bn(g);
     tbb::flow::limiter_node<int> ln(g, try_put_num);
-    tbb::flow::make_edge(bn, ln);
+#if __GNUC__ && __GNUC__ < 12 && !TBB_USE_DEBUG
```

Comment:
And do we really need the first check?

```suggestion
#if __GNUC__ < 12 && !TBB_USE_DEBUG
```